### DISCUSSION TITLE
Welcome to homebridge-platform-insteonlocal Discussions!

### DISCUSSION THREAD

**Discussion by kuestess** [kuestess/homebridge-platform-insteonlocal#222] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ctgreybeard** [comment]
Great idea!

Hi, I'm Bill Waggoner, an aging computer geek (1965 was my original introduction to computers) and I have used Insteon since it began as an improvement to X-10. I don't have much left but I'm thankful for this plugin to allow me to continue using what I have.

**Reply by JimCSaratoga** [reply]
This is great.  @kuestess thank you again for your great work here.  I've had Insteon since 2004  in my homes. Still amazed at the performance which IMHO still has not been surpassed.
